This file is part of MXE.
See index.html for further information.

Make sure exec_prefix is defined in wavpack.pc.

diff -ur a/wavpack.pc.in b/wavpack.pc.in
--- a/wavpack.pc.in	2013-09-29 23:30:14.000000000 +0200
+++ b/wavpack.pc.in	2013-11-17 18:31:35.351567596 +0100
@@ -1,4 +1,5 @@
 prefix=@prefix@
+exec_prefix=${prefix}
 libdir=@libdir@
 includedir=${prefix}/include
 
